iroha-lib/README.md

```diff
@@ -1,16 +1,118 @@
-# C++ library for Hyperledger Iroha
+# Hyperledger Iroha C++ library
 
 
-This is a source code for HL Iroha C++ library.
+### Current version of the library was tested and compatible with Iroha v2.0.0-pre.rc.6
```

Comment:
Incorrect version, Iroha 2 has different commands and queries.
```suggestion
### Current version of the library was tested and compatible with Iroha 1.5.0
```

Reply:
Ok